doc/source/containers.rst

```diff
@@ -116,6 +116,13 @@ container.
     >>> container.start()
     >>> container.freeze()
     >>> container.delete()
+   
+   
+Config line with a specific image source and a profile.
+
+.. code-block:: python
+
+    >>> config = {'name': 'somename', 'source': {'type': 'image', "mode": "pull", "server": "https://cloud-images.ubuntu.com/daily", "protocol": "simplestreams", 'alias': 'bionic/amd64'}, 'profiles': ['profilename'] }
```

Comment:
It would be better if this was pretty-printed (laid out nicely) from a docs perspective.